setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import codecs
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "allure2html"
+CMD = "allure2html"
```

Comment:
I think that "allure2html" is a quite ambigous name for package (and, so, for command), I would recommend "allure-combine" or "allure2singlehtml"

Reply:
OK, sounds great.